src/internal_modules/roc_address/protocol_map.cpp

```diff
@@ -129,5 +129,44 @@ void ProtocolMap::add_proto_(const ProtocolAttrs& proto) {
     protos_[proto.protocol] = proto;
 }
 
+bool ProtocolMap::get_supported_interfaces(core::Array<Interface>& interface_array) {
+    interface_array.clear();
+    bool interfaces_exist = false;
+
+    for (unsigned x = (unsigned)Iface_Consolidated + 1; x != (unsigned)Iface_Max; x++) {
+        for (size_t y = 0; y < MaxProtos; y++) {
```

Comment:
nitpick: here and in other places; let's rename counters like "x" and "y" to something a bit more meaningful so that they are not confused with each other.

E.g. "x" can be renamed to "n_iface" or "ni", and "y" to "n_proto" or "np", or something like this.